src/openforms/scss/components/_form-choices.scss

```
@@ -1,5 +1,5 @@
 @import 'microscope-sass/lib/bem';
-@import 'microscope-sass/lib/responsive';
+@import '~microscope-sass/lib/typography';
```

the ~ construct is a legacy thing and is no longer needed

```suggestion
@import 'microscope-sass/lib/typography';
```